src/components/SharedComponents/UploadStatus/UploadStatus.js

```diff
@@ -37,18 +37,19 @@ const UploadStatus = ( {
     rotateAnimation.setValue( 0 );
   } );
 
-  // const interpolateRotating = rotateAnimation.interpolate( {
-  //   inputRange: [0, 1],
-  //   outputRange: ["0deg", "360deg"]
-  // } );
+  const interpolateRotating = rotateAnimation.interpolate( {
+    inputRange: [0, 1],
+    outputRange: ["0deg", "360deg"]
+  } );
 
-  // const rotate = {
-  //   transform: [
-  //     {
-  //       rotate: interpolateRotating
-  //     }
-  //   ]
-  // };
+  const rotate = {
```

Comment:
Hi @angielt, I did not mean to step on your toes on this one, but I was checking out #571 and I _think_ it was caused by the following:

```
  Animated.loop(
    Animated.timing( rotateAnimation, {
      toValue: 1,
      duration: 10000,
      easing: Easing.linear,
      useNativeDriver: true
    } )
  ).start( () => {
    rotateAnimation.setValue( 0 );
  } );
  ```
  
 Not entirely sure though, as I couldn't repro the issue. In any case I make a draft PR switching out from React Native's animation library to reanimated. Reason being the spinning dots animation will most likely freeze once uploads start and the JS thread clogs. The other being to hopefully fix #571 

But you can take a look at that PR and if you decide to go with it, I don't mind and can close mine!

Reply:
No worries, I had thoughts of potentially switching to use the reanimated library. In this case I think #571 was because I forgot to add a stop animation call.

Reply:
@angielt ahh that makes sense